core/src/dird/fd_cmds.cc

```diff
@@ -127,7 +127,7 @@ static bool connect_outbound_to_file_daemon(JobControlRecord* jcr,
   utime_t heart_beat;
 
   if (!IsConnectingToClientAllowed(jcr)) {
-    Dmsg1(120, "connecting to client \"%s\" is not allowed.\n",
+    Dmsg1(120, "Connection from director to client \"%s\" is not allowed.\n",
```

Comment:
```suggestion
    Dmsg1(120, "Connecting to client \"%s\" is not allowed.\n",
```

Reply:
I would suggest to keep the wording unified (You added "Connecting to %s is not allowed" in line 231, so I would suggest to call it like this here also. Or we would need to update all other occurrences to contain the word "director" also.

Reply:
That's the only occurrence of `IsConnectingToClientAllowed(jcr)` where there is a message returned. Adding a `from director to client` makes sense in my opinion to specify the exact reason on that occasion. But `IsConnectFromClientAllowed(jcr)` has a few occurrences that I can easily update.

For line 231, I did not add a `from` since both types of connection would be closed. If the check is verified, there is no way to connect to the client at all.

Reply:
The "from director" is not needed as dmesg log entries are prefixed witht he daemon name that logs.